README.adoc

```diff
@@ -36,30 +38,238 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 
 [[initial]]
-== Create a resource controller
+== Add required dependencies
+
+To use the Spring Integration GCP Pub/Sub channel adapters, you'll need to pull in the
```

Comment:
I cut most of the text like you suggested. Although bear in mind that before anything, this guide really is a way of getting users started with the channel adapters. That's why you see them referred to so much.